iso15118/shared/settings.py

```diff
@@ -23,4 +23,6 @@
 )
 
 ENABLE_TLS_1_3 = env.bool("ENABLE_TLS_1_3", default=False)
+
+shared_settings = env.dump()
```

Comment:
One minor suggestion - would be good to call env.dump() after env.seal()?

Reply:
you're right. Done.